financialaid/views.py

```diff
@@ -224,23 +227,56 @@ class FinancialAidActionView(UpdateAPIView):
     queryset = FinancialAid.objects.all()
 
 
-class GetLearnerPriceForCourseView(APIView):
+class CoursePriceListView(APIView):
     """
-    View for retrieving a leaner's price for a course run
+    View for retrieving a learner's price for course runs in all enrolled programs
+    """
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        """
+        GET handler
+        """
+        user = request.user
+        program_enrollments = ProgramEnrollment.objects\
+            .select_related('user', 'program')\
+            .filter(user=user, program__live=True).all()
+        formatted_course_prices = []
+        for program_enrollment in program_enrollments:
+            try:
+                response_dict = get_formatted_course_price(program_enrollment)
+                response_dict['program_id'] = program_enrollment.program.id
+                formatted_course_prices.append(response_dict)
+            except ImproperlyConfigured:
```

Comment:
i'll need to be convinced of that. with the current code, a user calls the /course_prices endpoint to get a list of prices for all of their enrolled programs. if their enrolled programs don't have a course price set up, then i think it makes sense for that call to return an empty list. also worth noting is the fact that this would cause a ton of headaches for anyone running the app locally without the right data.